src/extra/loadmodule

```diff
@@ -1,6 +1,10 @@
 echo loadmodules
 
 module purge
-module load hdf5/1.8.11  
-module load netcdf-4.3.3.1-openmpi-intel
-module load git/2.1.2
+module load ncarenv/1.2
```

Comment:
Could you please revert these changes to `src/extra/loadmodule` as they are redundant when using the `src/extra/env/cheyenne` environment.